exporter/awsemfexporter/README.md

```diff
@@ -14,8 +14,8 @@ The following exporter configuration parameters are supported.
 
 | Name              | Description                                                            | Default |
 | :---------------- | :--------------------------------------------------------------------- | ------- |
-| `log_group_name`  | Customized log group name                                              |"/metrics/default"|
-| `log_stream_name` | Customized log stream name                                             |"otel-stream"|
+| `log_group_name`  | Customized log group name which supports `{{ClusterName}}` and `{{TaskId}}` placeholders. One valid example is `/aws/metrics/{{ClusterName}}`. It will search for `ClusterName` (or `aws.ecs.cluster.name`) resource attribute in the metrics data and replace with the actual cluster name. If none of them are found in the resource attribute map, `{{ClusterName}}` will be replaced by `undefined`. Similar way, for the `{{TaskId}}`, it searches for `TaskId` (or `aws.ecs.task.id`) key in the resource attribute map.                                           |"/metrics/default"|
```

Comment:
```suggestion
| `log_group_name`  | Customized log group name which supports `{{ClusterName}}` and `{{TaskId}}` placeholders. One valid example is `/aws/metrics/{{ClusterName}}`. It will search for `aws.ecs.cluster.name` resource attribute in the metrics data and replace with the actual cluster name. If none of them are found in the resource attribute map, `{{ClusterName}}` will be replaced by `undefined`. Similarly, for the `{{TaskId}}`, it searches for `aws.ecs.task.id` key in the resource attribute map.                                           |"/metrics/default"|
```

Reply:
Here, It will search for `ClusterName` first. If not found then it will search for the default semantic convention `aws.ecs.cluster.name`. We chose this because some ECS customers rename their resource attributes with short names before data comes to the exporter. So, we need to check both.